packages/vitest/src/types/config.ts

```diff
@@ -120,6 +123,8 @@ interface DepsOptions {
    * This could be helpful to handle packages that ship `.js` in ESM format (that Node can't handle).
    *
    * If `true`, every dependency will be inlined
+   *
+   * @deprecated If you rely on vite-node directly, use `server.deps.inline` instead. Otherwise, consider using `deps.optimizer.web.include`.
```

Comment:
```suggestion
   * @deprecated If you rely on vite-node directly, use `server.deps.inline` instead. Otherwise, consider using `deps.optimizer.{web,ssr}.include`.
```